src/main/java/strategy/StoreOperationsHandler.java

```diff
@@ -0,0 +1,7 @@
+package strategy;
+
+import model.FruitTransaction;
+
+public interface StoreOperationsHandler {
```

Comment:
```suggestion
public interface OperationsHandler {
```
same in another cases